gremlin-javascript/src/main/javascript/gremlin-javascript/lib/structure/graph.ts

```diff
@@ -73,20 +77,26 @@ export class Vertex<T extends Record<string, any> = Record<string, any>> extends
   }
 }
 
-export class Edge<T extends Record<string, any> = Record<string, any>> extends Element {
+export class Edge<
+  TOutVertex extends Vertex = Vertex,
```

Comment:
Why `TOutVertex extends Vertex`? Why not simply `Vertex`?

Reply:
The extends part will restrict the input type to `Vertex`, while the `= Vertex` part will default that generic to a generic Vertex if no type is provided.